### PR TITLE
Update information-barriers-in-teams.md

### PR DESCRIPTION
**Pull request by Vikramju**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -97,19 +85,12 @@ Coming soon: users will experience the following if an information barrier polic
...
 
-Information barriers are rolling out now, and are included in subscriptions, such as:
-
-- Microsoft 365 E5
-- Office 365 E5
-- Office 365 Advanced Compliance
-- Microsoft 365 E5 Compliance
-
-For more details, including plans and pricing, see [Compliance Solutions](https://products.office.com/business/security-and-compliance/compliance-solutions?rtc=1).
+For more details, including plans and pricing, see [Licensing Guidance](https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance).
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md]:
```suggestion
For more details, including plans and pricing, see [Microsoft 365 tennant-level services licensing guidance](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance).
```

File: Teams/information-barriers-in-teams.md [hunk 2]
```diff
@@ -97,19 +85,12 @@ Coming soon: users will experience the following if an information barrier polic
...
 
 ## More information
 
 - To learn more about information barriers, see [Information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers).
 
 - To set up information barrier policies, see [Define policies for information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers-policies)
 
-- To edit or remove information barrier policies, see [Edit (or remove) information barrier policies](https://docs.microsoft.com/office365/securitycompliance/information-barriers-edit-segments-policies.md)
\ No newline at end of file
+- To edit or remove information barrier policies, see [Edit (or remove) information barrier policies](https://docs.microsoft.com/office365/securitycompliance/information-barriers-edit-segments-policies.md)
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md]:
```suggestion
- To edit or remove information barrier policies, see [Edit (or remove) information barrier policies](https://docs.microsoft.com/microsoft-365/compliance/information-barriers-edit-segments-policies.md).
```

File: Teams/information-barriers-in-teams.md [hunk 3]
```diff
@@ -97,19 +85,12 @@ Coming soon: users will experience the following if an information barrier polic
...
-- Microsoft 365 E5 Compliance
-
-For more details, including plans and pricing, see [Compliance Solutions](https://products.office.com/business/security-and-compliance/compliance-solutions?rtc=1).
+For more details, including plans and pricing, see [Licensing Guidance](https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance).
 
 ## More information
 
 - To learn more about information barriers, see [Information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers).
 
 - To set up information barrier policies, see [Define policies for information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers-policies)
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md]:
```suggestion
- To set up information barrier policies, see [Define policies for information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers-policies).
```

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**